nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
When the user/pass is passed locally in the options, then the credentials are included.  Updates from an async INFO will not have any credentials.  Shall I strip them out?  (The application can read the credentials elsewhere)